database/migrations/2016_02_17_175810_create_competition_user_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateCompetitionUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('competition_user', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('competition_id')->unsigned()->index();
+            $table->foreign('competition_id')->references('id')->on('competitions');
```

Comment:
@sbsmith86 chatted further with @DFurnes and think you're correct on being concerned it might delete a user when competition is deleted. I was kind of following the examples with some old code on Aurora, but turns out that isn't even used anymore. Removed the `->onDelete('cascade')` so I can look into it further, and might be that those methods need to be on the individual migration tables and not on the pivots. Will keep you posted once I learn more ;)